model/ftn/ww3_grid.ftn

```diff
@@ -6276,6 +6290,7 @@
                           READ (NDS,NML=OUTS,END=801,ERR=802,IOSTAT=J)
                         CASE('MISC')
                           READ (NDS,NML=MISC,END=801,ERR=802,IOSTAT=J)
+                          IF ( CAL360 .AND. NOLEAP ) GOTO 805
```

Comment:
See comment at line 2589

Reply:
Agree with the change of location for the check on CAL360 and NOLEAP. Waiting for suggestions on where to add the 360-day calendar regtest.

When using namelist files, the calendar is read from the file specified by the parameter NML_GRID%NML. In the template file this is pointing to the file 'namelist.nml', which does not exist.

Reply:
namelist.nml exists for each regtest

Reply:
What I meant is that the namelist.nml template file does not exist. But of course, in the new regtest to check the 360-day calendar the CAL360 parameter is properly set. In the rest of the regtests it is not included because its default value is FALSE as expected.

Reply:
ok, so we can create a file namelist.nml in the model/nml which contains some namelist values if you also want

Reply:
I can do it, adding all the possible namelist parameters with their default value.

Reply:
Thanks Juan, the new version using CALTYPE works for this